### PR TITLE
[zh-cn] sync network-plugins labels

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md
content/zh-cn/docs/concepts/overview/working-with-objects/labels.md

### CODE REVIEW

File: content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md [hunk 1]
```diff
@@ -16,12 +16,14 @@ weight: 10
...
 <!-- 
-Kubernetes {{< skew currentVersion >}} supports [Container Network Interface](https://github.com/containernetworking/cni)
+Kubernetes (version 1.3 through to the latest {{< skew latestVersion >}}, and likely onwards) lets you use
+[Container Network Interface](https://github.com/containernetworking/cni)
 (CNI) plugins for cluster networking. You must use a CNI plugin that is compatible with your
 cluster and that suits your needs. Different plugins are available (both open- and closed- source)
 in the wider Kubernetes ecosystem.
 -->
-Kubernetes {{< skew currentVersion >}} 支持用于集群联网的[容器网络接口](https://github.com/containernetworking/cni)（CNI）插件。
+Kubernetes（1.3 版本至最新 {{< skew latestVersion >}}，并可能继续更新）
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md:25]:
```suggestion
Kubernetes（1.3 版本至最新 {{< skew latestVersion >}}，并可能包括未来版本）
```

File: content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md [hunk 2]
```diff
@@ -16,12 +16,14 @@ weight: 10
...
+要求使用用于集群联网的[容器网络接口](https://github.com/containernetworking/cni)（CNI）插件。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md:26]:
```suggestion
允许你使用[容器网络接口](https://github.com/containernetworking/cni)（CNI
插件来完成集群联网。
```

**Review by mengjiao-liu — COMMENTED** [review]
/lgtm

**Pull request merged**